ci/Jenkinsfile.android

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.9.24'
+
+/* Options section can't access functions in objects. */
+def isPRBuild = utils.isPRBuild()
+def isNightlyBuild = utils.isNightlyBuild()
+
+pipeline {
+  agent {
+    /* Image with Ubuntu 22.04, QT 6.9.0, Android SDK/NDK, Go, and Nim */
+    docker {
+      label 'linux'
+      image 'statusteam/nim-status-client-build:1.0.0-qt6.9.0-android-ci'
+      alwaysPull true
+    }
+  }
+
+  parameters {
+    booleanParam(
+      name: 'RELEASE',
+      description: 'Decides whether release credentials are used.',
+      defaultValue: params.RELEASE ?: false
+    )
+  }
+
+  options {
+    timestamps()
+    /* Prevent Jenkins jobs from running forever */
+    timeout(time: 45, unit: 'MINUTES')
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '10',
+      daysToKeepStr: '30',
+      artifactNumToKeepStr: '3',
+    ))
+    /* Allows combined build to copy */
+    copyArtifactPermission('/status-desktop/*')
+    /* Abort old PR builds. */
+    disableConcurrentBuilds(abortPrevious: isPRBuild)
+    disableRestartFromStage()
+  }
+
+  environment {
+    USE_SYSTEM_NIM = "1"
+    GOCACHE = "/tmp/go-cache"
+    GOMODCACHE = "/tmp/go-mod-cache"
+    GOTMPDIR = "/tmp"
+    STATUS_APK_ARTIFACT = "pkg/Status-tablet-${BUILD_NUMBER}.apk"
+    PLATFORM = "android/arm64"
+    QT_ANDROID_PATH = "/opt/qt/6.9.0/android_arm64_v8a"
+    STATUS_APK = "${WORKSPACE}/mobile/bin/android/qt6/Status-tablet.apk"
+  }
+
+  stages {
+    stage('Prepare') {
+      steps {
+        sh 'git submodule update --init --recursive'
+        echo "Building for architecture: arm64"
+        echo "Qt Android path: ${env.QT_ANDROID_PATH}"
+      }
+    }
+
+    stage('Build Android APK') {
+      steps {
+        sh """
+          # These exports MUST be in the build stage because the build system uses qmake to detect OS
+          # The correct Android qmake must be first in PATH to return 'android-clang' instead of 'linux-g++'
```

Comment:
That's not true. You can set those in `environment` just fine.

Reply:
I tried but the build would fail with this error : 
https://github.com/status-im/status-desktop/blob/2ad002891a6b69e3ba89225c814d1af8bf52431c/mobile/scripts/EnvVariables.mk#L54

most probably because of this logic here : 
https://github.com/status-im/status-desktop/blob/2ad002891a6b69e3ba89225c814d1af8bf52431c/mobile/scripts/EnvVariables.mk#L1-L15

Reply:
this happens : https://ci.status.im/blue/organizations/jenkins/status-desktop%2Fprs%2Fandroid%2Farm64%2Fpackage/detail/PR-18597/30/pipeline#step-48-log-5

Reply:
Did you actually try to print the env vars? They seem correct to me but maybe some path was missing.